UM/Application.py

```diff
@@ -101,6 +101,12 @@ def __init__(self, name: str, version: str, build_type: str = "", **kwargs):
 
         self._operation_stack = OperationStack(self.getController())
 
+        # Ensure configuration folder has a folder for themes
+        try:
+            os.makedirs(os.path.join(Resources.getStoragePath(Resources.Resources), "themes"))
```

Comment:
Ah, I see... The folder does not get created because ```Themes``` is not in ```Resource.__types_storage```. Which would be more preferable, creating the folder manually, or insinuating ```themes``` is a storage folder?